### PR TITLE
Add support for MySQL cursor

### PR DESCRIPTION
**Pull request by BillyYccc**
fix https://github.com/eclipse-vertx/vertx-sql-client/issues/281.

### CODE REVIEW

**Comment by vietj** [comment]
`ExecutionMode` seems MySQL specific and it should not be in the shared classes, can you move this to the MySQL specific implementation ?

**Comment by BillyYccc** [comment]
> ExecutionMode seems MySQL specific and it should not be in the shared classes, can you move this to the MySQL specific implementation ?

Updated with https://github.com/eclipse-vertx/vertx-sql-client/pull/289/commits/4cd74fa1b52698a2f5427d1da38f3d72ed61f99d.

File: src/main/java/io/vertx/sqlclient/impl/command/CloseStatementCommand.java [hunk 1]
```diff
@@ -17,9 +17,19 @@
 
 package io.vertx.sqlclient.impl.command;
 
+import io.vertx.sqlclient.impl.PreparedStatement;
+
 /**
  * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
  */
 public class CloseStatementCommand extends CommandBase<Void> {
+  private final PreparedStatement ps;
```
Comment (vietj) [inline src/main/java/io/vertx/sqlclient/impl/command/CloseStatementCommand.java]: rename to `statement`

File: src/main/java/io/vertx/sqlclient/impl/command/ClosePortalCommand.java [hunk 2]
```diff
@@ -17,19 +17,26 @@
...
 
+import io.vertx.sqlclient.impl.PreparedStatement;
+
 /**
  * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
  */
 public class ClosePortalCommand extends CommandBase<Void> {
 
   private final String portal;
+  private final PreparedStatement ps;
```
Comment (vietj) [inline src/main/java/io/vertx/sqlclient/impl/command/ClosePortalCommand.java]: rename to `statement`

**Comment by vietj** [comment]
I'll rename myself

**Pull request merged**